### DISCUSSION TITLE
Manual / Tutorial

### DISCUSSION THREAD

**Discussion by Switch-Witch** [terjeio/ioSender#377] [General]
Hi, I am new to this fantastic App but for me as a beginner it is more than hard to get into it without any kind of manual/tutorial.
Ist there any kind of this available?

best

Kai

**Comment by phil-barrett** [comment]
[This](https://www.grbl.org/single-post/one-sender-to-rule-them-all) is getting a bit old but still applicable.

**Comment by terjeio** [comment]
There are also some [videos by Stefan Bauer](https://www.youtube.com/playlist?list=PLnSV6o2cRxM5mQQe4ec5cS2J8jBsEciY3) available from the _Help_ menu,
